src/tests/test_api/mysql.rs

```diff
@@ -39,6 +39,17 @@ impl<'a> MySql<'a> {
 
         Ok(Self { names, conn, tag })
     }
+
+    fn render_perm_create_table(&mut self, table_name: &str, columns: &str) -> (String, String) {
```

Comment:
Do we btw. clean these tables after a test is run, so they don't leak to the other tests?